backend/dataall/cdkproxy/stacks/policies/aws_cicd.py

```diff
@@ -65,18 +96,23 @@ def get_statements(self):
                 },
             ),
             iam.PolicyStatement(
-                actions=['codebuild:*'],
+                #sid="CodePipelineCreateS3Bucket",
+                effect=iam.Effect.ALLOW,
+                actions=[
+                    's3:CreateBucket',
+                    's3:ListBucket',
+                    's3:PutBucketPublicAccessBlock',
+                    's3:GetObject',
+                    's3:PutObject',
+                    's3:DeleteObject'
+                ],
                 resources=[
-                    f'arn:aws:codebuild:{self.region}:{self.account}:project/{self.resource_prefix}*',
-                    f'arn:aws:codebuild:{self.region}:{self.account}:report-group/{self.resource_prefix}*',
+                    "arn:aws:s3:::codepipeline-*",
```

Comment:
who creates these buckets? these resources should probably be scoped down more. do the buckets have account/region identifiers in their names? something to make this more specific?

Reply:
Yes, we can! Thank you!